internal/cephfs/nodeserver.go

```diff
@@ -118,6 +119,40 @@ func validateSnapshotBackedVolCapability(volCap *csi.VolumeCapability) error {
 	return nil
 }
 
+// maybeUnlockFileEncryption unlocks fscrypt on stagingTargetPath, iff volOptions enable encryption.
+func maybeUnlockFileEncryption(
```

Comment:
Any specific reason to start with `maybe` for both functions?

Reply:
Both functions only do things if encryption is enabled. The 'maybe' tried to convey that the function may not do anything at all.